lean4-infoview/src/infoview/interactiveCode.tsx

```diff
@@ -58,6 +58,14 @@ const LazyTippy = React.forwardRef<HTMLElement, TippyProps>((props, ref) => {
     computedProps.content = mounted ? props.content : undefined
   }
 
+  if (computedProps.appendTo == 'parent'){
+    // need to go some levels higher because we sometimes have nested spans and so parent is
+    // not enough to avoid weird layout bugs where Chrome does not like a <div> popping up
+    // in the middle of a string of spans, even though the div is position:absolute, it is
+    // causing newlines to appear (bug https://github.com/leanprover/vscode-lean4/issues/51)
+
```

Comment:
This is an empty if?  (The `appendTo=` property below would also be a good place for the comment.)

Reply:
woops, yeah, dead code sorry.  removed.

Reply:
Could you please add the comment back somewhere?  It's good to know what the document.body setting is for.

Reply:
sure thing, done.